lib/java/benchmarks-common/src/main/java/org/enso/interpreter/bench/BenchmarksRunner.java

```diff
@@ -107,4 +117,76 @@ private static BenchmarkItem reportResult(String label, RunResult result) throws
     Report.writeToFile(report, REPORT_FILE);
     return benchItem;
   }
+
+  private static final class GitHubActionsFormat implements OutputFormat {
+    private final OutputFormat output;
+
+    GitHubActionsFormat(OutputFormat output) {
+      this.output = output;
+    }
+
+    @Override
+    public void iteration(BenchmarkParams benchParams, IterationParams params, int iteration) {
+      output.iteration(benchParams, params, iteration);
+    }
+
+    @Override
+    public void iterationResult(
+        BenchmarkParams benchParams, IterationParams params, int iteration, IterationResult data) {
+      output.iterationResult(benchParams, params, iteration, data);
+    }
+
+    @Override
+    public void startBenchmark(BenchmarkParams benchParams) {
+      output.println("::group::" + benchParams.getBenchmark());
```

Comment:
The `::group::` format seems to be mentioned at [this discussion](https://github.com/orgs/community/discussions/25618).

Reply:
It is documented here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#grouping-log-lines

Although the documentation is not very detailed.